src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,358 @@
+
+                // Set mode to use custom delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings |= AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            }
+            else
+            {
+                // Set mode to use default delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings &= ~AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            }
+
+            // Kick it off with a single acquire command.
+            WriteRegister(Register.ACQ_COMMAND, 0x04);
+        }
+
+        /// <summary>
+        /// Set a new I2C address.
+        /// </summary>
+        /// <remarks>
+        /// Note, if the device is powered off or reset, the IC2 address will reset to the default address.
+        /// </remarks>
+        /// <param name="address">new address, valid values are 7-bit values with 0 in the LSB.</param>
+        public void SetI2CAddress(byte address)
+        {
+            if ((address & 1) == 1)
+            {
+                throw new ArgumentOutOfRangeException("Address must have 0-bit in the LSB.");
+            }
+
+            // Read in the unit's serial number.
+            Span<byte> rawData = stackalloc byte[2];
+
+            ReadBytes(Register.UNIT_ID, rawData);
+
+            // Write serial number to I2C_ID.
+            WriteRegister(Register.I2C_ID_HIGH, rawData[1]);
+            WriteRegister(Register.I2C_ID_LOW, rawData[0]);
+
+            // Write the new address.
+            WriteRegister(Register.I2C_SEC_ADDR, address);
+
+            // Disable the default address
+            WriteRegister(Register.I2C_CONFIG, 0x08);
+        }
+
+        #region Device Registers
+
+        /// <summary>
+        /// Get the last distance measurement in cm.
+        /// </summary>
+        public int LastDistance
+        {
+            get
+            {
+                Span<byte> rawData = stackalloc byte[2];
+                ReadBytes(Register.FULL_DELAY, rawData);
+                return BinaryPrimitives.ReadUInt16BigEndian(rawData);
+            }
+        }
+
+        /// <summary>
+        /// Get the difference between the current and last measurement resulting
+        /// in a signed (2's complement) 8-bit number in cm.
+        /// Positive is away from the device.
+        /// </summary>
+        public int DifferenceBetweenLastTwoDistances
+        {
+            get
+            {
+                Span<byte> rawData = stackalloc byte[1];
+                ReadBytes(Register.VELOCITY, rawData);
+                return (int)(sbyte)rawData[0];
+            }
+        }
+
+        /// <summary>
+        /// Get or set the various settings to control the acquistion behavior.
+        /// </summary>
+        public AcquistionSettings AcquistionSettings
```

Comment:
Looks like you also have a typo on the word Acquisition here, it would be probably good to just do a find all instances of where you have it misspelled and fix them.

Reply:
Fixed all instances of this.